src/framework/preferences/css/stylus/utils/Helpers.styl

```diff
@@ -21,3 +21,23 @@ linear-gradient() {
         s('linear-gradient(%s)', content);
     }
 }
+
+
+// Mixin for common styles of the "thumb" (slider control)
+// used by both the jQuery and native HTML sliders
+slider-thumb-common() {
+    height: 1.5em;
+    width: 1.5em;
+    background-color: #fff;
+    border: 1px solid #b3b3b3;
+    border-radius: 2em;
+    box-shadow: 4px 2px 3px rgba(0,0,0,0.3);
```

Comment:
@jhung yes, this is just a straight migration of the existing styles for the jQuery-based sliders, with the common characteristics between styling native & jQuery pulled into the helper

I assume we'd want the border and box-shadow to scale via `em`?